### DISCUSSION TITLE
issue missing

### DISCUSSION THREAD

**Discussion by FuPeiJiang** [mark-wiemer/ahkpp#63] [General]
@mark-wiemer did you delete an issue I made about keyword/bracket in multiline string
https://github.com/mark-wiemer/vscode-autohotkey-plus-plus/issues/41 : doesn't exist anymore

I needed this snippet and couldn't find it.
```autohotkey
exampleString1=
(
multiple 
lines {
    of 
    string
    )
```
if you've deleted it:
You could've told me and I could've renamed it or reworded it.
but please don't permanently delete when working with others/

quote:
Aurelio Ogliari
I see why this was done, but it looks like the wrong way to fix the problem. Maintaining the full conversation history of a repo just goes along well with the philosophy of git, and this feature goes against this idea. A lot of information to be lost this way.
___
I'll try to keep things on my repo

**Comment by mark-wiemer** [comment]
No, I didn't delete your issue. I didn't know you could delete issues until I looked around just now! This is strange indeed, I'm sorry your issue is lost

**Reply by mark-wiemer** [reply]
Dang, even Wayback Machine doesn't have anything for us to reference ([Wayback Machine - github.com/mark-wiemer](https://web.archive.org/web/20210304162818/https://github.com/mark-wiemer)). I'm really sorry about this, I've never intentionally deleted an issue and I don't know how this happened. Please open another issue if you'd like me to look into this

**Comment by FuPeiJiang** [comment]
sorry, I know you didn't delete the issue: I tried deleting an issue and it gives [this page](https://github.com/FuPeiJiang/vscode-autohotkey-plus-plus/issues/2) instead of 404 :
`This issue has been deleted.`<br>
![](https://i.stack.imgur.com/pmIQ5.png)


I think it's my fault: I didn't create the issue, but I thought I did.
it's improbable that the browser failed to send.
even more that github deleted it.
___

I found this : https://jira.atlassian.com/browse/JRASERVER-12909
github may be similar ?
this may explain the cause of gaps in issue #Number.

**Reply by mark-wiemer** [reply]
No worries. It is curious that there is a missing issue number though... oh well. Not sure if the JIRA issue is relevant since GitHub and JIRA are separate platforms, but there may be a similar (faulty) pattern that both have implemented
